getting-started/README.md

```diff
@@ -69,7 +69,7 @@ $ npm install \
 
 ([link to TypeScript version](ts-example/README.md#initialize-a-global-tracer))
 
-All tracing initialization should happen before your application’s code runs. The easiest way to do this is to initialize tracing in a separate file that is required using node’s `-r` option before application code runs.
+All tracing initialization should happen before your application code runs. The easiest way to do this is to initialize tracing in a separate file that is required using NodeJS's `-r` option before your application code runs.
```

Comment:
it is in fact `node`'s `-r` option, not NodeJS per se, `node` is the CLI tool and the `-r` option comes from it. You could put backticks around `node` though

Reply:
Fixed.